contrib/chatops/CHANGES.md

```diff
@@ -1,8 +1,12 @@
 # Changelog
 
```

Comment:
Can you please also add a CHANGELOG entry into the root CHANGELOG.rst so it's known to a wider audience and can be captured in the release notes for StackStorm?

Reply:
Will do

Reply:
@nzlosh still need changelog in root CHANGELOG.rst